source/_integrations/lamarzocco.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: La Marzocco
+description: Instructions on how to integrate your La Marzocco coffee machine with Home Assistant.
+ha_release: 2023.10
+ha_category:
+  - Water Heater
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_domain: lamarzocco
+ha_platforms:
+  - water_heater
+ha_codeowners:
+  - '@zweckj'
+  - '@rccoleman'
+ha_integration_type: integration
+---
+
+This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through a mix of cloud calls, local API calls, WebSocket connections and (optionally) Bluetooth. 
+
+If you are in Bluetooth reach to your coffee machine it should be discovered automatically.
+
+Your machine needs to be added to your account using the official app first.
```

Comment:
```suggestion
If you are in Bluetooth reach to your coffee machine it should be discovered automatically.

Your machine needs to be added to your account using the official app first.
```
Not sure I understand this correctly. The second sentence here, should it go first? 
First, I have to add the machine to the account in the official app, and then, after I added the integration to Home Assistant, the coffee machine is detected by the integration in Home Assistant?

Reply:
It should be discovered anyways. However, to be able to go through the setup you need to it to your account first.